docs/snippets/MULTI4.cpp

```diff
@@ -1,19 +1,19 @@
-#include <ie_core.hpp>
+#include <openvino/openvino.hpp>
 
 int main() {
-const std::map<std::string, std::string> hddl_config  = { { InferenceEngine::PluginConfigParams::KEY_PERF_COUNT, InferenceEngine::PluginConfigParams::YES } };
-const std::map<std::string, std::string> gpu_config = { { InferenceEngine::PluginConfigParams::KEY_PERF_COUNT, InferenceEngine::PluginConfigParams::YES } };
+ov::AnyMap hddl_config  = {{ov::enable_profiling(true)}};
+ov::AnyMap gpu_config = {{ov::enable_profiling(true)}};
 //! [part4]
 // configure the HDDL device first
-InferenceEngine::Core ie; 
-InferenceEngine::CNNNetwork cnnNetwork = ie.ReadNetwork("sample.xml");
-ie.SetConfig(hddl_config, "HDDL"); 
-// configure the GPU device
-ie.SetConfig(gpu_config, "GPU"); 
+ov::Core core;
+std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
+core.set_property({ov::device::properties("HDDL", hddl_config),
```

Comment:
Can we demonstrate here that `ov::device::properties` for HDDL and GPU as passed to compile_model directly? In this case global state for devices is not changed.
And update the description in the document itself.

@apankratovantonp does this code even work? I mean ov::device::properties with set_property?

Reply:
Yes it should work. It will dothe same as:
```c++
 core.set_property("HDDL", hddl_config);
 core.set_property("GPU", gpu_config);
```
 
 But `{` and `}` can be dropped:
```c++
core.set_property(
    ov::device::properties("HDDL", hddl_config),
    ov::device::properties("GPU", gpu_config));
```

Reply:
@ilya-lavrenov  there is much way to set config,  if comile_model(mode,device, ov::device::properties1,ov::device::properties2) is the recommend way to do it for openvion API 2.0, I will update it in snippets

Reply:
fixed